.github/workflows/docs.yml

```diff
@@ -0,0 +1,58 @@
+name: Build Documentation
+
+on:
+  workflow_call:
+    inputs:
+      version:
+        description: "The version string for this release"
+        required: true
+        type: string
+      pdf-name:
+        description: "The name of the PDF file generated by latexpdf, as set in your conf.py"
+        required: true
+        type: string
+      python-version:
+        description: "The version of Python used to build the docs, defaults to 3.8.10"
+        default: 3.8.10
+        required: false
+        type: string
+      dependencies:
+        description: "The Python dependencies to build the docs, defaults to Sphinx==3.5.4"
+        default: "Sphinx==3.5.4"
```

Comment:
Should be required we need at less Sphinx dependency. 
Suggestion: Add "space separated list of python dependencies" in the description.

Reply:
Same comment as above